### ISSUE TITLE
1% accuracy lower than expectation with default config

### ISSUE THREAD

**Issue by ShawnHSH** [ZJCV/ZCls#10]
Hi,

I'm new to this fantastic work. When I tried to test the inference accuracy of resnet-18 with r18_torchvision_imagenet_224.yaml/r18_zcls_imagenet_224.yaml, the result accuracy is 68.136%, which is lower than your reported result, 69.22%. I wonder if I need to modify the config file or anything else? 

Really looking forwards to your reply! Thanks a lot for your contribution to the open-source community!

**Comment by zjykzj** [comment]
> Hi,
> 
> I'm new to this fantastic work. When I tried to test the inference accuracy of resnet-18 with r18_torchvision_imagenet_224.yaml/r18_zcls_imagenet_224.yaml, the result accuracy is 68.136%, which is lower than your reported result, 69.22%. I wonder if I need to modify the config file or anything else?
> 
> Really looking forwards to your reply! Thanks a lot for your contribution to the open-source community!

Hi @HaroldHong,  ZCls has been iterated through many versions at present. I haven't tested all the test files in every round, and I can't answer your question because of the lack of detailed operation process.. By the way, r18 training weight is converted from torchvision pre-training model.

In addition, I am developing [ZCls2](https://github.com/ZJCV/ZCls2), a classification framework with faster training speed.  If you want to train classification tasks, I recommend you to use this framework.

**Comment by ShawnHSH** [comment]
Thanks for your quick reply! I have only used ZCLs for inferencing remote pretrained models, i.e. I did not modify the code files or config files. I installed the zcls-v0.15.2 from the source, and I also tried versions 0.15.0 and 0.14.0. But the inference accuracy is still 68.136%. 

I am so excited and looking forward to the ZCls2. Thanks again for your contributions!
